### PR TITLE
I added my component eyeMap in home.

### PR DESCRIPTION
**Pull request by Sonya092**

### CODE REVIEW

File: src/components/EyeMap/style.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const SEye = styled.main`
```
Comment (KeerhemFR) [inline src/components/EyeMap/style.jsx]: Better to use another tag, main usually wrapping a lot of elements it could cause conflict here.

**Review by KeerhemFR — DISMISSED** [review]
One little evil, but can be fix later

File: src/components/EyeMap/index.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  };
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    setDetails(texts[e.target.alt]);
+  };
+  return (
+    <SEye>
+      <img src={ALJschemas} useMap="#oeil-map" alt="Imgoeil" />
+      <map name="oeil-map">
```
Comment (Hacene010) [inline src/components/EyeMap/index.jsx]: no french plz

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Saw a small evil plz fix it.

**Review by KeerhemFR — APPROVED** [review]
See no more evil

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**